testcase.py

```diff
@@ -97,6 +99,8 @@ def __initilize_data(self):
         for key in self.input_names:
             self.u[key] = np.array([])
         self.u_store = copy.deepcopy(self.u)
+        # Results filtering for pyfmi
+        self.options['filter'] = self.output_names + self.input_names
```

Comment:
Should the filter option be set in the __init__ with the other options?  This just may be more consistent to have all the options set in the __init__ when they can be (basically move this to after line 60, and move call of __initialize_data accordingly in __init__).

Reply:
I think that totally makes sense. Implemented in https://github.com/ibpsa/project1-boptest/pull/324/commits/b9110650339c09bf2e80c00cbb7d04b711760a41.